homeassistant/components/energyzero/services.yaml

```diff
@@ -1,5 +1,10 @@
 get_gas_prices:
   fields:
+    config_entry:
```

Comment:
What does this config entry?

Reply:
It allows to select the config entry for an integration.

If only one config entry can exist for the integration an alternative is to pick the first config entry for the integration domain. Then the user doesn't need to specify it. But if we ever could allow more than one config entry in the future for the integration, it could be good to include this parameter from the beginning so that we don't need to break the service later.

Reply:
I would rather not have users to search for the config entry ID to use it in the template sensor.  Does not benefit the UX

Reply:
In the frontend there's a nice selector so the user can just pick it from a list.

Reply:
I've added it because I could not find another way to load the coordinator from an entity and saw all other examples of services use it. Please let me know how you want to have this.

Reply:
But what if you use yaml code? Don't think we want people to have to dig through their .storage folder for an ID

Reply:
I agree that it is not really user friendly. Who would decide on something like this?

Reply:
Maybe this is a solution: https://www.home-assistant.io/docs/configuration/templating/#config-entries

That we retrieve the config entry ID of an energyzero entity in this way? Instead of the long ID string that we now have in the sensor template 🤔

Reply:
> But what if you use yaml code? Don't think we want people to have to dig through their .storage folder for an ID

It's quite easy to switch back and forth between GUI and YAML in the automation editor.

Reply:
I will also implement everything and try some things with easyEnergy. I understand that it is easy for GUI users, but we should not forget the hardcode YAML users either (like me 😄). Everyone is going to make those template sensors so that they can put the data in Apexcharts, so maybe we should document this.

Reply:
The easiest way for users to find the config entry ID is to first try out the service in the UI and then switch to YAML, then the config entry ID will become visible automatically.

Reply:
Yeah, that's what I was meaning in my comment above. 👍